testing/ansible-tester/Dockerfile

```diff
@@ -1,29 +1,69 @@
-FROM williamyeh/ansible:centos7
+FROM ubuntu:22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
 
 ### Update system packages
-RUN yum --assumeyes --quiet upgrade
+RUN apt-get -q -y update
+RUN apt-get -q -y upgrade
+
+RUN apt-get install -q -y \
+    dmidecode \
+    python3 \
+    python3-pip \
+    gpg-agent \
+    rpm  \
+    openssh-client \
+    wget
 
-### Install additional ansible support
-RUN yum --assumeyes --quiet install dmidecode python3 rpm-build wget
 # XXX - The latest version of pip (21.3.1) is breaking on CentOS 7 without
 #       following encoding setup.
-ENV LANG=en_US.UTF-8
+ ENV LANG=en_US.UTF-8
 # XXX - ^^^
+
+# XXX - To resolve ansible.utils.display.initialize_locale (unsupported locale setting).
+ ENV LC_ALL=C
+ # XXX - ^^^
```

Comment:
Is this a bug in ansible.utils.display.initialize_locale, or is it a requirement for the function to work correctly?  If it's a requirement, use the `NB` instead of `XXX`.

Reply:
It's a requirement, not a bug.. I will use NB.
Note:
#11 [ 7/52] RUN locale -a
C
C.utf8
 POSIX

Reply:
It's a requirement, not a bug.. I will use NB.
Note:
#11 [ 7/52] RUN locale -a
C
C.utf8
 POSIX